_data/alumni.yml

```diff
@@ -60,6 +60,22 @@
       icon: github-alt
       name: Github
 
+- name: Amanda Bachmann
+  pic: /img/people/bachmanna.png
+  dates: 2020-present
+  role: Graduate Research Assistant
+  description: "Amanda is a PhD student interested in the nuclear fuel cycle and the best ways to implement nonproliferation safeguards."
+  social:
+      - url: mailto:amandab7@illinois.edu
+        icon: envelope-o
+        name: email
```

Comment:
```suggestion

```

Reply:
What is this changing?

Reply:
As far as I can tell I'll lose access to this email 1 year after graduation, so maybe not the best to have that there.